src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -306,6 +308,15 @@ subroutine step_MOM_dyn_split_RK2(u, v, h, tv, visc, Time_local, dt, forces, p_s
     ! u_old_rad_OBC and v_old_rad_OBC are the starting velocities, which are
     ! saved for use in the Flather open boundary condition code [L T-1 ~> m s-1].
 
+  real, dimension(SZIB_(G),SZJ_(G),SZK_(G)) :: hf_PFu ! Zonal Pressure force accel. x fract. thickness [L T-2 ~> m s-2].
+  real, dimension(SZI_(G),SZJB_(G),SZK_(G)) :: hf_PFv ! Merdional Pressure force accel. x fract. thickness [L T-2 ~> m s-2].
+  real, dimension(SZIB_(G),SZJ_(G),SZK_(G)) :: hf_CFu ! Zonal Coriolis force accel. x fract. thickness [L T-2 ~> m s-2].
+  real, dimension(SZI_(G),SZJB_(G),SZK_(G)) :: hf_CFv ! Merdional Coriolis force accel. x fract. thickness [L T-2 ~> m s-2].
+
+  real, dimension(SZIB_(G),SZJ_(G),SZK_(G)) :: hfrac_u
```

Comment:
I'd suggest naming these variables something like `diag_hfrac_u` and `diag_hfrac_v` since they are not used prognostically.